### DISCUSSION TITLE
Model Element Author specification using IDS

### DISCUSSION THREAD

**Discussion by femesq** [buildingSMART/IDS#340] [Q&A] [answered]
When creating a BIM model, we have the [Model Element Author](https://bimdictionary.com/en/model-element-author/1) - which defines who is responsible for modeling each element.

Initially, this verification seemed to be viable using the entities [IfcOwnerHistory](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcOwnerHistory.htm) and [IfcPerson](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcPerson.htm) / IfcOrganization. But reading the documentation (and examples), I couldn't understand how this would be possible to go down this deep relation. Eg.:

`IfcWall` -> `IfcOwnerHistory (attribute)` -> `OwningUser (attribute)` - type: [IfcPersonAndOrganization](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcPersonAndOrganization.htm) -> `IfcPerson | IfcOrganization`

Is this possible to achieve currently?

____

When moving to requirements specification, this could be even more granular and allow a third party to be responsible for providing such information, although someone else is the modeler (who populates the property/attribute in the model).

Is it possible to define who is responsible for meeting this requirement using the IDS, or because it is impossible to check (since the IFC will not have such data), is this outside the scope of the IDS?

**Comment by mysterieux30** [comment]
I think not yet

**Comment by CBenghi** [comment, marked as answer]
This is out of scope for version 1.0. It cannot be done with IDS.
You could do this with MVDXML, I think.
